components/UI/ListItem.tsx

```diff
@@ -0,0 +1,23 @@
+import React, {
+  DetailedHTMLProps,
+  LiHTMLAttributes,
+  PropsWithChildren,
+} from 'react';
+
+interface Props
+  extends PropsWithChildren<
+    DetailedHTMLProps<LiHTMLAttributes<HTMLLIElement>, HTMLLIElement>
+  > {
+  right?: React.ReactNode;
+}
+
+const ListItem = ({ right, children, ...props }: Props) => {
+  return (
+    <li {...props}>
+      {children}
+      {right}
```

Comment:
What does "right" mean? Isn't that included in `children`?

Reply:
right means literally component will be placed in right side